azure-iot-device/azure/iot/device/iothub/pipeline/config.py

```diff
@@ -22,6 +30,7 @@ def __init__(self, hostname, device_id, module_id=None, product_info="", **kwarg
         :param str device_id: The device identity being used with the IoTHub
         :param str module_id: The module identity being used with the IoTHub
         :param str product_info: A custom identification string for the type of device connecting to Azure IoT Hub.
+        :param bool ensure_desired_properties: A boolean to determine whether or not to refresh desired properties in the EnsureDesiredPropertiesStage (Default:True)
```

Comment:
Again, don't need to specify type since it's already there. In this case, since this is an internal class you also don't need to mention the default value since it's obvious from the definition.

Adapt this to the phrasing style of connection_retry and auto_connect parameters on the base config.